app/build.gradle

```diff
@@ -56,24 +55,19 @@ dependencies {
     implementation 'com.google.firebase:firebase-analytics-ktx'
     implementation 'com.google.firebase:firebase-auth-ktx'
     implementation 'com.google.firebase:firebase-database-ktx'
+    implementation 'com.google.firebase:firebase-firestore-ktx'
     implementation "com.xwray:groupie:2.9.0"
+    implementation 'com.android.support.test.espresso:espresso-idling-resource:3.0.2'
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.4.1'
-    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.0"
```

Comment:
I would be careful in deleting so many dependencies, are you sure these are not needed somewhere else in the project?

Reply:
I updated it.

Reply:
My b, I must have not rebased correctly, I had a lot of git problems.